### DISCUSSION TITLE
Vega lite - Labels and background colour

### DISCUSSION THREAD

**Discussion by GeorgeRobbin** [vega/vega-lite#8619] [Q&A] [answered]
1 - Im attempting to create a chart which displays 4 sections based on values on each axis (-x,-y)(-x,y)(-y,x)(xy), is there a way to set a separate background colour for each section?
2 - is there a way to add a permanent label to specific data points (text function i believe)
[HOUSE_MORTGAGE.json](https://github.com/GeorgeRobbin/GeorgeRobbin.github.io/blob/main/HOUSE_MORTGAGE.json)

**Comment by jwoLondon** [comment, marked as answer]
The key here is to use [layers](https://vega.github.io/vega-lite/docs/layer.html) – one for your background quadrants, one for the dots and one for the labels. Here's [one approach](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAKyUAVhDYA7EABoQAdxoATemjEAGbUvg0yWNAGZdSwUgCecAE5oA2qDVJMKVKGJJBDOBEegAB5o-GZKgQBMaHogViEAjGJKVlGoMbJICHBoILZwaiAAvgpB0eGpAJwxcWnJqRHpmdnoTD7ZxaWooTGRZbF9KWgNShlZOWR5cPIdIMG1s5XVCUmxqY1j6DaCgmzKRQC6xSBTUGxqNLJkaJ2gAGY0cIIF6MFKmFY4zSAAjgxIspg6K4aKQiuVriB7o9ngswf0PJCHk8cnE3h8vr9-oC3IDQUdBgiocjNlEjqcdvZCUiYaNskpoN5mrIGNtCkdkLYANYQ96fHJ5KCYRQgNg4JBQOg1bSUCJskogFxuCEMWyCHJYXD4Ii2JDKah0RhMBgQOynAFTTCUU4IAgAcTgbFsZDgACU2EwmBc7Q6na73Z7ZPr6MxWBxkF6AMIwJC2TARAD6AAkAAoASQApCYAIIAWRdVogxDhJzOFyuCLmd2pORzjswFGdAAIXa44I2s7Jmd5G+nG1H-s7hdjBF9a7GG22W5g2x2u4Ie33o5c6SBeRi-gCgbiV0hAjR-AiJuo0LdvCajgyRxCAF52Ngns9wOXwqvQnKJtjGtvJ2w0KAzzs-nnXt+2XIc6CvdAPy-Rsfz-AC5wXUDBzRPl0ExTccRBHc9wPUAj2eU9BHPekoEZG87wfYin2fDluQRNccglWwoCvJRyUdfl8mFUVxUlaJKAAdjlZxXHcTxvF8A8nBABBKQANn0HAEkoEwLCQJhHhyAAZDTHkbeIwVAOSEmlFYYGU1B4kUkBLE0tV0F0+zG1JQ4lBLc5LghStETfdATNQ9csS3bC4RqV9iQwZTApyDDsWBPE3jgQIhSpPzbL0tVaJjejQEY9BpxS4U1DmAAWJRvEMeR0BHW4hSUJgkBNYRZC+BB1DUK82UOIA) that uses the [rect](https://vega.github.io/vega-lite/docs/rect.html) mark to draw the quadrants each with a custom colour determined by inline data.


<img width="585" alt="quads" src="https://user-images.githubusercontent.com/1846999/210168174-2599c7de-4946-420b-a4e5-9580a24747ac.png">
